.github/workflows/lua.yml

```diff
@@ -42,9 +42,10 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v3
-    - name: Install luarocks
-      run: |
-        sudo apt-get update && sudo apt-get install -y luarocks
+    - uses: leafo/gh-actions-lua@v9
+      with:
+        luaVersion: "5.1.5"
```

Comment:
I think you can skip this and install lua from the repos (`lua5.1`)

Reply:
Why would that be preferable?

Reply:
might be faster, less third-party dependencies, ...

Reply:
Doesn't work; the luarocks action presumably depends on the Lua action.

Reply:
smh